packages/babel-preset-react-app/create.js

```diff
@@ -79,11 +67,8 @@ module.exports = function(api, opts, env) {
         // Latest stable ECMAScript features
         require('@babel/preset-env').default,
         {
-          // We want Create React App to be IE 9 compatible until React itself
-          // no longer works with IE 9
-          targets: {
-            ie: 9,
-          },
+          // FamilySearch compatibility target, not necessarily React's target:
+          targets: '> 0.5% and not dead and not ie <= 10',
```

Comment:
are we going to keep ie11 just because we can? I was looking forward to working on more older browsers just help out a few patrons, but we can go full supported list and add `last 2 versions` and exclude IE entirely.
I'm in favor of keeping some older browser support around if it's easy to do, have you compared bundles sizes with IE removed?

Reply:
Our current override also has `not op_mini all`

Reply:
I'm glad you are bringing this up. There's a tradeoff: If we include the older browsers, we will increase our bundle size. We can also make two versions of our bundles, one for older browsers and one for more modern browsers, as explained [here](https://babeljs.io/docs/en/babel-preset-env#targetsesmodules).